test/e2e_node/remote/node_e2e.go

```diff
@@ -65,6 +73,18 @@ func (n *NodeE2ERemote) SetupTestPackage(tardir string) error {
 		}
 	}
 
+	if systemSpecName != "" {
+		// Copy system spec file
+		source := filepath.Join(rootDir, systemSpecPath, systemSpecName+".yaml")
+		if _, err := os.Stat(source); err != nil {
+			return fmt.Errorf("failed to locate system spec %q: %v", source, err)
+		}
+		out, err := exec.Command("cp", source, filepath.Join(tardir, "system-spec.yaml")).CombinedOutput()
```

Comment:
Can we copy the system spec directly into tar without renaming? Seems less code and less confusing. :)